README.md

```diff
@@ -16,6 +16,7 @@ python main.py --common_config <common_config_name> --model <model_name> --model
 | dataset | pa_lov, ... | Dataset to use |
 | fold | 0, 1, ... | Fold number |
 | mode | fit, predict, fit_predict | Fit, predict or fit and predict together |
+| gpu | 0, 1, ... | Physical GPU ID |
```

Comment:
Please add this argument in the command above as well.

Reply:
Added the GPU flag in the example. Also added the optional flag for the gpu argument in the arguments table.